project-base/tests/ShopBundle/Acceptance/acceptance/PageObject/Admin/ProductEditPage.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin;
+
+use Tests\ShopBundle\Acceptance\acceptance\PageObject\AbstractPage;
+
+class ProductEditPage extends AbstractPage
+{
+    /**
+     * @param int $productId
+     */
+    public function saveInvalidForm($productId)
+    {
+        $this->tester->amOnPage('/admin/product/edit/' . $productId);
+        // scroll to have the button visible because of the fixed bars
```

Comment:
is it necessary to scroll down? I thought that when you do just `$this->tester->clickByCss('.js-parameters-item-add');`, it does not matter that the element is not in viewport, or does it?

Reply:
Yes, there is the problem with fixed elements. Click method will do the scroll to element like function ```javascript
document.querySelector('.js-parameters-item-add').scrollIntoView()
```
and then client will try to click on it but there is element fixed bar above it and codeception will fail on error 
```
Element \<a href="#" class="js-parameters-item-add btn btn--plus">...\</a> is not clickable at point (338, 15). Other element would receive the click: \<header class="web__header">...\</header>
```